services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy

```diff
@@ -0,0 +1,73 @@
+package org.openkilda.functionaltests.spec.resilience
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.HeartBeat
+import org.openkilda.messaging.Message
+import org.openkilda.messaging.ctrl.KafkaBreakTarget
+import org.openkilda.messaging.info.event.IslChangeType
+import org.openkilda.testing.model.topology.TopologyDefinition
+import org.openkilda.testing.service.kafka.KafkaBreaker
+
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+
+import java.util.concurrent.TimeUnit
+
+class FloodlightKafkaConnectionSpec extends BaseSpecification {
+    @Autowired
+    KafkaBreaker kafkaBreaker
+    @Autowired
+    TopologyDefinition topology
+
+    @Autowired
+    @Qualifier("kafkaConsumerProperties")
+    Properties consumerProps
+    @Value('${kafka.topic.topo.disco}')
+    String topic
+
+    def "System survives temporary connection outage between Floodlight and Kafka"() {
+        when: "Controller loses connection to kafka"
+        kafkaBreaker.shutoff(KafkaBreakTarget.FLOODLIGHT_PRODUCER)
+        kafkaBreaker.shutoff(KafkaBreakTarget.FLOODLIGHT_CONSUMER)
+
+        and: "Remains in this state for 12 seconds"
```

Comment:
Why 12??? Let's move this to constants and leave a small comment about what is that time.

Reply:
12 was used in original test, just copied the value..
Changed to 10 seconds and added a comment about why we use this timeout. I do not want to move this to Constants, because this is not related to any global constant. This is a local 'test' value and I don't see any reason to move to separate global class in different maven module